### PR TITLE
one_click.py

### PR DESCRIPTION
**Pull request by teufortressIndustries**
* FEAT: Added a bat file to simplify launching the script from the source.

I'm creating a one-click installer to simplify the process of the initial install.
The installer will detect the user's operating system and execute the required commands.

### CODE REVIEW

**Comment by dborodin836** [comment]
Thanks for the PR! This will be useful, especially for new users. I'll take a look and ensure compatibility across different operating systems. As far as I remember, there may be some issues with a Python package on platforms other than Windows, or maybe I'm just hallucinating.

**Comment by dborodin836** [comment]
Yeah, it seems that there are a few issues related to the stats functionality, stopping/starting the bot and the GUI.

File: start.sh
Comment (dborodin836) [inline start.sh:1]: Remove that file

File: start.bat
Comment (dborodin836) [inline start.bat:1]: Rename the file to be more obvious for users `start.bat` -> `start_windows.bat`

File: one_click.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "pip install -r requirements.txt",
+        "python main.py"
+    ]
+else:
+    print("Unsupported operating system")
+    exit()
+
+# execute the commands in a shell
+for command in commands:
+    subprocess.run(command, shell=True)
```
Comment (dborodin836) [inline one_click.py:28]: That section doesn't seem to work on my system. Please try replacing it with the suggested code, delete the `venv` directory, and run the batch file again.
```
# execute the commands in a shell
subprocess.run(' & '.join(commands), shell=True)
```

File: one_click.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+os_name = sys.platform
+
+# run the commands depending on the os
+if os_name.startswith("linux"):
+    commands = [
+        "python3 -m venv venv",
+        "source venv/scripts/activate",
+        "pip install -r requirements.txt",
+        "python main.py"
+    ]
```
Comment (dborodin836) [inline one_click.py:14]: Remove that section

**Review by dborodin836 — CHANGES_REQUESTED** [review]
Since the Linux version doesn't work properly, let's leave only the Windows .bat script and `one_click.py`. So, please remove any Linux-related stuff.

File: one_click.py (hunk 2 above)
Comment (teufortressIndustries) [reply one_click.py]: Removed

**Review by dborodin836 — APPROVED** [review]
Looks good to me

**Pull request merged**